draft-ietf-privacypass-protocol.md

```diff
@@ -332,15 +332,12 @@ The values `token_input` and `blinded_element` are stored locally and used
 later as described in {{private-finalize}}. The Client then generates an HTTP
 POST request to send to the Issuer Request URL, with the TokenRequest as the
 content. The media type for this request is
-"application/private-token-request". An example request is shown below.
+"application/private-token-request". An example request for the Issuer Request URL
+"https://issuer.example.net/request" is shown below.
 
 ~~~
-:method = POST
```

Comment:
I think you also need a Host header here

Reply:
Fixed